site/index.html

```diff
@@ -200,6 +200,7 @@
         ],
       };
     </script>
+    <script src="//kpt.dev/static/js/plugins/copy_buttons.js"></script>

```

Comment:
I'm curious why moving this to the front will help. Can you please briefly explain it a bit?

Reply:
The local docsify plugin code (like the configuration above) needs to be imported/defined before docsify initializationhttps://github.com/GoogleContainerTools/kpt-functions-catalog/pull/378/files/0fa5fb0109c11825b7199da5fec752bd4332e7c7#diff-aef9cf1e2772f3835b32360da3eb558ecc9624843d3a91fe6a2c7e64b7a2c5e6R204
Otherwise, the plugins are attached but not registered.